src/content/docs/data-apis/manage-data/query-limits.mdx

```diff
@@ -141,15 +157,20 @@ From left to right, top to bottom:
     id=""
     title="Limit list and NrIntegrationError"
   >
-    ```
-    FROM Metric, NrIntegrationError select rate(sum(newrelic.resourceConsumption.currentValue), 1 minute) as 'Per Minute Count',latest(newrelic.resourceConsumption.limitValue) as ' limit Value',(rate(sum(newrelic.resourceConsumption.currentValue), 1 minute)/latest(newrelic.resourceConsumption.limitValue)*100)as 'Percent Used', filter (count(*), where NrIntegrationError.limitValue is not null) as 'limit reached count' facet limitName limit 1000
+    ```sql
+    FROM Metric, NrIntegrationError 
+    SELECT rate(sum(newrelic.resourceConsumption.currentValue), 1 minute) AS 'Per Minute Count',
+      latest(newrelic.resourceConsumption.limitValue) AS 'Limit Value',
+      (rate(sum(newrelic.resourceConsumption.currentValue), 1 minute) / latest(newrelic.resourceConsumption.limitValue) * 100) AS 'Percent Used', 
+      filter(count(*), WHERE NrIntegrationError.limitValue is not null) AS 'Limit Reached Count' 
+    FACET limitName LIMIT 1000
     ```
   </Collapser>
 </CollapserGroup>
 
 ## Limit metrics
 
-These metrics, used in the dashboard queries above, can hone in on a single limit or resource. Or, with the help of `FACET limitName or resource` provide a view across all your limits.
+These metrics, used in the dashboard queries above, can hone in on a single limit or resource. Or, with the help of `FACET limitName OR resource` provide a view across all your limits.
```

Comment:
Suggestion: These metrics, used in the dashboard queries above, can focus on a single limit or resource. Alternatively, using `FACET limitName OR resource` gives a view across all your limits.